docs/get_started.rst

```diff
@@ -8,6 +8,17 @@ Overview
 .. automodule:: django_select2
     :members:
 
+Assumptions
+-----------
+
+* You have a Django project, possibly with models linked by foreign key fields,
+* are using Python3,
+* set up a virtual environment with ``python3 -m venv <location>``,
+* activated the virtual environment,
+* installed ``django`` using ``pip``,
```

Comment:
Django is a dependency of this package. So you don't necessarily need to install it. However, you do need an app setup.
Also, you may use `pipenv`, `pip` is not hard requirement.

Reply:
I hadn't ever heard of `pipenv`, but I read »The official recommended way [to install pipenv] is to use pip«.  

assumption: »you have a Django app«?

Reply:
`Pipenv` is a whole toolset. You can use it to describe the entire environment including checksums, to ensure your environments are the same across machines. You should try it out!

Besides that, maybe better: you have a **running** Django  app